### ISSUE TITLE
Unhandled Exception

### ISSUE THREAD

**Issue by alexfalkowski** [fsharp-editing/Forge#89]
Hi,

Running the command `forge` by itself in the command line produces the following

```
Unhandled Exception:
System.IndexOutOfRangeException: Index was outside the bounds of the array.
  at Forge.Commands.applyAlias (System.String[] args) [0x00007] in <59b9172273406ffea74503832217b959>:0
  at Forge.Commands.runForge (System.String[] args) [0x00001] in <59b9172273406ffea74503832217b959>:0
  at Forge.App.main (System.String[] argv) [0x0007b] in <59b9172273406ffea74503832217b959>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.IndexOutOfRangeException: Index was outside the bounds of the array.
  at Forge.Commands.applyAlias (System.String[] args) [0x00007] in <59b9172273406ffea74503832217b959>:0
  at Forge.Commands.runForge (System.String[] args) [0x00001] in <59b9172273406ffea74503832217b959>:0
  at Forge.App.main (System.String[] argv) [0x0007b] in <59b9172273406ffea74503832217b959>:0
```

**Comment by Krzysztof-Cieslak** [comment]
Fixed in 36c9f5e8decbc7bf6e5b569a1feaa7db2aa800e0, will be available in next release

**Comment by AkosLukacs** [comment]
Hi! Maybe it's just me, because 2.0.0 is not yet available on scoop, so just overwritten the binaries, but without arguments, it does blow up:
```
>forge

Unhandled Exception: System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Forge.Commands.applyAlias(String[] args)
   at Forge.Commands.runForge(String[] args)
   at Forge.App.main(String[] argv)
```

Win 10, Forge 2.0.0 downloaded from the 2.0.0 release zip.
